ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -676,6 +682,41 @@ impl fmt::Display for PtrAddInstruction {
     }
 }
 
+/// The operands for a [Instruction::Add]
```

Comment:
pedantry (that i'd only fix if you have other changes to make):
 - *an* InstructionAdd
 - blank docstring line at end

Reply:
We'll have to update that in all the other descriptions too.

Reply:
I leave it up to you.